_layouts/product-overview.html.slim

```diff
@@ -50,6 +53,11 @@ br
     div(class="feature-text")
       = page._features3
 
+- if page._special_table
```

Comment:
Why can't you put the table in the 'detail' section, like we do for Fuse: https://github.com/redhat-developer/developers.redhat.com/blob/master/products/fuse/overview.adoc#detail